### ISSUE TITLE
Can we consume it in Azure Devops ?

### ISSUE THREAD

**Issue by himanshush13** [SAP/devops-cm-client#56]

**Comment by 1084565** [comment]
Yes, you can fetch it from [maven central](https://mvnrepository.com/artifact/com.sap.devops.cmclient) or, recommended, use the [Docker image](https://registry.hub.docker.com/r/ppiper/cm-client). Keep in mind, connectivity from Azure to your on-prem system (ABAP or SAP Solution Manager) may be a challenge. Anything to add @radsoulbeard ?
